index.js

```diff
@@ -1,22 +1,29 @@
-const myLibrary = [];
-
 const formContainer = document.getElementById('form_container');
 const bookShelf = document.getElementById('book_shelf');
+class Book {
+
+  static myLibrary = []
+  static bookID = 0
 
-let bookID = myLibrary.length - 1;
-function Book(title, pages, author) {
-  this.id = bookID;
-  bookID += 1;
+  constructor(title, pages, author, read = false) {
+    this.id = Book.bookID;
+    this.title = title;
+    this.pages = pages;
+    this.author = author;
+    this.read = read;
 
-  this.title = title;
-  this.pages = pages;
-  this.author = author;
-  this.read = false;
+    Book.myLibrary.push(this);
```

Comment:
The push to the library can be its own `class method`
Think about moving it out of the `constructor`